packages/webpack/src/runtime/runtime.js

```diff
@@ -27,6 +34,11 @@ if (LOCKDOWN_ON) {
   )
 }
 
+// harden appears on globalThis only after lockdown is called
+const {
+  harden,
+} = globalThis
```

Comment:
`harden` isn't skipped over by the scuttling process, so without these lines, it is being scuttled (see  https://github.com/LavaMoat/LavaMoat/blob/main/packages/core/src/scuttle.js#L132).

We can add `harden` to that list, or cache it here, but without both a later use of `harden` encounters a scuttled property instead of the expected value of the `harden` property of the globalThis (here https://github.com/LavaMoat/LavaMoat/blob/main/packages/webpack/src/runtime/runtime.js#L247).